fem/bilinearform.cpp

```diff
@@ -27,9 +27,19 @@ void BilinearForm::AllocMat()
       return;
    }
 
-   const Table &elem_dof = fes->GetElementToDofTable();
+   Table elem_dof(fes->GetElementToDofTable());
```

Comment:
I see that the vast majority of the changes are isolated in the templated code, which is great. Out of curiosity, why does BilinearForm have to change?

Reply:
This is fixing the seg fault when using UsePrecomputedSparsity() for ND and RT element. It is not for partial assembly.